app/controllers/colleges_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class CollegesController < ApplicationController
+  include AdminOnlyAccess
```

Comment:
The Colleges page should have been restricted from regular user access.  Its only purpose is to list, change, and destroy colleges.